src/main/java/com/ait/lienzo/client/core/shape/wires/handlers/WiresParentPickerControl.java

```diff
@@ -8,4 +8,13 @@
     public WiresContainer getParent();
 
     public PickerPart.ShapePart getParentShapePart();
+
+    public Index getIndex();
+
+    public interface Index {
+
+        public void addShapeToSkip(WiresContainer shape);
```

Comment:
maybe a javadoc on the interface/method to detail what is this Index used for and skip the shape from what?

Reply:
Yeah that Index interface was not good at all, good catch thanks! Just refactored and added some javadocs,... :+1: